olive/auto_optimizer/workflow_template/ort_perf_tuning.yaml

```diff
@@ -0,0 +1,17 @@
+input_model:
+  type: "ONNXModel"
+  model_path: <<model_path>>
+passes:
+  perf_tuning:
+    type: "OrtPerfTuning"
+    data_config:
```

Comment:
@shaahji brought up that we don't want to support having data config dicts passed directly to metrics or passes anymore in a workflow config. 
In order to avoid conflicts in the future, can you define this in data_configs and refer to the name like in our examples?

Reply:
Updated.